df.ui-menus.xml

```diff
@@ -987,13 +987,25 @@
             abstractitemlistst master_a_list;
 -->
 
-                <stl-vector/>
-                <stl-vector type-name='int16_t'/>
-                <stl-vector type-name='int16_t'/>
-                <stl-vector type-name='int16_t'/>
-                <stl-vector type-name='int16_t'/>
-                <stl-vector type-name='int32_t'/>
-                <stl-bit-vector/>
+                <stl-vector name='unk_a8'>
```

Comment:
this is the vector I'd probably be accessing in the tool I'm working on. Could it possibly get a slightly more permanent name? like `item_data` or something?

Reply:
i'd like to have slightly more of an idea of what is in there before i give it even a semi-permanent name

keeping it as an `unk` signals that "this is not suitable for use in user-facing code" which it totally is not at this point